request.proto

```diff
@@ -113,13 +113,14 @@ message JourneysRequest {
 }
 
 message PlacesNearbyRequest {
-    required string uri         = 1;
-    required double distance    = 2;
-    repeated NavitiaType types  = 3;
-    required int32 depth        = 4;
-    required int32 count        = 5;
-    required int32 start_page   = 6;
-    optional string filter      = 7;
+    required string uri                   = 1;
+    required double distance              = 2;
+    repeated NavitiaType types            = 3;
+    required int32 depth                  = 4;
+    required int32 count                  = 5;
+    required int32 start_page             = 6;
+    optional string filter                = 7;
+    optional bool find_stop_points_nearby = 8;
```

Comment:
What do you think about default valur ? -  '[default=false]'

Reply:
It is **optional**. So, we managed it in an other part of code
just here https://github.com/CanalTP/navitia/pull/3201/files#diff-04e391bedc0175c758164a98a2915889R506